system_metrics_collector/CMakeLists.txt

```diff
@@ -48,7 +49,7 @@ add_library(system_metrics_collector SHARED
   src/system_metrics_collector/proc_pid_cpu_data.cpp
   src/system_metrics_collector/utilities.cpp
 )
-ament_target_dependencies(system_metrics_collector metrics_statistics_msgs rclcpp rcpputils rcutils)
+ament_target_dependencies(system_metrics_collector metrics_statistics_msgs rclcpp rclcpp_lifecycle rcpputils rcutils)
```

Comment:
Consider breaking that line as this starts getting too long.

Reply:
Will do.